hadoop-ozone/dev-support/checks/hadolint.sh

```diff
@@ -17,25 +17,24 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 REPO_DIR="$DIR/../../.."
 
-ERROR=0
-
-
-for Dockerfile in $(find "$REPO_DIR/hadoop-ozone" "$REPO_DIR/hadoop-hdds" -name Dockerfile | sort); do
-  echo "Checking $Dockerfile"
+REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../target/hadolint"}
+mkdir -p "$REPORT_DIR"
+REPORT_FILE="$REPORT_DIR/summary.txt"
```

Comment:
To make sure `hadolint.sh` can be run multiple times and still give correct results, rather than keep accumulating findings:

```suggestion
REPORT_FILE="$REPORT_DIR/summary.txt"
echo -n > "$REPORT_FILE"
```